game/missiongenerator/aircraft/flightgroupspawner.py

```diff
@@ -519,6 +520,18 @@ def _generate_at_cp_ground_spawn(
         group.points[0].type = "TakeOffGround"
         group.units[0].heading = ground_spawn[0].units[0].heading
 
+        if (
+            cp.coalition.game.settings.ground_start_airbase_statics_farps_remove
+            and isinstance(cp, Airfield)
+        ):
+            # Remove invisible FARPs from airfields because they are unnecessary
```

Comment:
But why exactly? 😅 
And why is almost the same code being executed again at the end of the method? 😅

Reply:
Because you cannot rearm close to these invisible FARPs unless they have their own separate fuel and ammo trucks/statics. Which degrade performance.

Reply:
refactored 👍